src/Lachain.Crypto/TransactionUtils.cs

```diff
@@ -26,36 +26,67 @@ public static int ChainId(bool useNewId)
             return useNewId ? _newChainId : _oldChainId;
         }
         
-        public static TransactionChainId GetEthTx(this Transaction t, Signature? s, bool useNewId)
+        public static RLPSigner GetEthTx(this Transaction t, Signature? s, bool useNewId)
         {
             var nonce = t.Nonce == 0
                 ? Array.Empty<byte>()
                 : new BigInteger(t.Nonce).ToByteArray().Reverse().ToArray().TrimLeadingZeros();
             var sig = s is null  ?  Array.Empty<byte>() : s.Encode().AsSpan();
-            var ethTx = new Nethereum.Signer.TransactionChainId(
+            var rlpSigner = new Nethereum.Signer.RLPSigner(new byte[][]{
                 nonce,
                 new BigInteger(t.GasPrice).ToByteArray().Reverse().ToArray().TrimLeadingZeros(),
                 new BigInteger(t.GasLimit).ToByteArray().Reverse().ToArray().TrimLeadingZeros(),
                 t.To.ToBytes(), // this may be empty, same as passing null
                 t.Value.ToBytes(false,  true),
                 t.Invocation.ToArray(),
-                new BigInteger(ChainId(useNewId)).ToByteArray().Reverse().ToArray().TrimLeadingZeros(),
+                new BigInteger(ChainId(useNewId)).ToByteArray().Reverse().ToArray().TrimLeadingZeros(), null, null},
                 sig.IsEmpty ? Array.Empty<byte>() : sig.Slice(0, 32).ToArray().TrimLeadingZeros(),
                 sig.IsEmpty ? Array.Empty<byte>() : sig.Slice(32, 32).ToArray().TrimLeadingZeros(),
-                sig.IsEmpty ? Array.Empty<byte>() : sig.Slice(64, sig.Length - 64).ToArray().TrimLeadingZeros()
+                sig.IsEmpty ? Array.Empty<byte>() : sig.Slice(64, sig.Length - 64).ToArray().TrimLeadingZeros(), 
+                6
```

Comment:
Not sure why using 6 here, is it the size of input byte[][] array? I see 9 elements including 2 null input. So should it be 9 instead of 6? Also maybe we can use a constant instead of number because it is used in other methods also.

Reply:
Nethereum doesn't use this param really in this case,  it uses constant value,  but we can set any number here really,

Reply:
I'll add a comment here